CHANGELOG.rst

```diff
@@ -7,7 +7,10 @@ This document describes changes between each past release.
 9.1.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+**Internal changes**
+
+- Refactor of ``kinto.core.storage.Storage.get_all`` method to issue two separate,
```

Comment:
nit: it only applies to PostgreSQL ;)

And also, I think it would make sense to mention the performance gain!